region/crateria/east.json

```diff
@@ -2662,6 +2662,153 @@
             {
               "id": 2,
               "strats": [
+                {
+                  "name": "Speedy Springball Jump and SpaceJump",
+                  "notable": false,
+                  "requires": [
+                    "canTrickyDashJump",
+                    "h_canUseSpringBall",
+                    "canMockball",
+                    "SpaceJump"
+                  ],
+                  "note": [
+                    "Run for 7 tiles for enough speed to springball from the submerged platform up to the platform enemy, above the water.",
+                    "It is not necessary to enter the water to clear the rightmost Choot."
+                  ]
+                },
+                {
+                  "name": "East Ocean Speedy Springball Hops and Bounce to the Door",
+                  "notable": true,
+                  "requires": [
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 15
+                    }},
+                    "canTrickyDashJump",
+                    "h_canUseSpringBall",
+                    "canDoubleSpringBallJumpMidAir",
+                    "canMockball"
+                  ],
+                  "note": [
+                    "Using an exact runway size of about 20 tiles; use extremely precise, controlled springball bounces to cross the ocean.",
+                    "Samus must bounce on the far right floor of the ocean with enough height to reach the waterline, without HiJump.",
+                    "While the bounce is occuring, setup a mid-air SpringBall Jump to escape the water and reach the door."
+                  ]
+                },
+                {
+                  "name": "Full Room Shinespark Plus SpaceJump",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        {"enemyDamage": {
+                          "enemy": "Choot",
+                          "type": "contact",
+                          "hits": 1
+                        }},
+                        "canNeutralDamageBoost",
+                        "canBounceBall"
+                      ]},
+                      {"and": [
+                        {"or": [
+                          "canPseudoScrew",
+                          "ScrewAttack",
```

Comment:
The choot is going to jump up into the screw attack, so that's fine.  Just gotta press mockball between hitting the enemy and touching the ramp.

But I'm seeing there are many more ways to activate this shinespark in time so I'll rewrite the strat.